controls/nist_rhcos4.yml

```diff
@@ -1759,6 +1759,10 @@ controls:
   rules:
   - wireless_disable_in_bios
   - kernel_module_bluetooth_disabled
+  - kernel_module_wireless_iwlmvm_disabled
+  - kernel_module_wireless_iwlwifi_disabled
```

Comment:
That's a good catch, I remember I had to remove the wireless, because I used the yaml template, and the template's oval definition name does not include wireless, I forgot to remove wireless from here